plugins/vivisect-broker/broker/vivserver.proto

```diff
@@ -8,12 +8,18 @@ message RunInstructionsRequest {
     int32 numInstructions = 1;
 }
 
+message MemoryAccess {
```

Comment:
`vivserver_pb2.*` are out of sync with this. You need to renegenerate the protobuf/grpc code.